test/unit-test/ota_os_posix_utest.c

```diff
@@ -153,15 +152,15 @@ void test_OTA_posix_TimerCreateAndStop( void )
 {
     OtaErr_t result = OTA_ERR_UNINITIALIZED;
 
-    result = timer.start( timer.PTimerCtx, TIMER_NAME, OTA_DEFAULT_TIMEOUT, NULL );
+    result = timer.start( timer_id, TIMER_NAME, OTA_DEFAULT_TIMEOUT, NULL );
     TEST_ASSERT_EQUAL( OTA_ERR_NONE, result );
 
     TEST_ASSERT_NOT_EQUAL( 0, getTimeElapsed() );
 
-    result = timer.stop( timer.PTimerCtx );
+    result = timer.stop( timer_id );
     TEST_ASSERT_EQUAL( OTA_ERR_NONE, result );
 
-    result = timer.delete( timer.PTimerCtx );
+    result = timer.delete( timer_id );
     TEST_ASSERT_EQUAL( OTA_ERR_NONE, result );
 }
 
```

Comment:
Doxygen briefs are interchanged for test_OTA_posix_TimerCreateAndStop() and test_OTA_posix_InvalidTimerOperations()